Introductory Tests/UDP_basic.ipynb

```diff
@@ -0,0 +1,1402 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #75.            return ([5000,5000,5000,10,10,10],[15000,15000,15000,5000,5000,5000])
</pre><p>these bounds don't make sense as the mesh is center around 0,0,0. Practically for r you will want something like -15k to 15k then. (Let's not worry about colliding with 67P for the moment)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rasmusmarak/67P-Swarm/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>